src/assets/i18n/en.json

```diff
@@ -57,5 +57,10 @@
   "DISCOVERED_BY_ME":"Discovered by me",
   "WEEKLY":"Weekly",
   "MONTHLY":"Monthly",
-  "QUARTERLY":"Quarterly"
-} 
\ No newline at end of file
+  "QUARTERLY":"Quarterly",
+  "DOWNLOADS":"Downloads",
+  "DELETE":"Delete",
+  "DELETE_CONTENT_HEAD":"Delete Content ?",
```

Comment:
Asked u to change the key right
Why is this not changed?

Reply:
done